source/configuration_manual/authentication/user_database_extra_fields.rst

```diff
@@ -22,6 +22,10 @@ fields'. Possibilities are:
   for returning a variable number of fields (e.g. SQL).
 * ``uidgid_file``: Get uid and gid for user based on the given filename.
 
+* ``event_<name>=value``: Import ``name=value`` to mail user event.
```

Comment:
The first =value part is inconsistent both with how this list works and how it was in the passdb list, so just remove it.